src/main/java/jsr223/docker/compose/utils/DockerComposePropertyLoader.java

```diff
@@ -46,20 +46,20 @@
 public class DockerComposePropertyLoader {
 
     @lombok.Getter
-    public String dockerHost;
+    private final String dockerHost;
 
     @lombok.Getter
-    public String dockerComposeCommand;
+    private final String dockerComposeCommand;
 
     @lombok.Getter
-    public String sudoCommand;
+    private final String sudoCommand;
 
     @lombok.Getter
-    public boolean useSudo;
+    private final boolean useSudo;
 
-    protected Properties properties;
+    private final Properties properties;
 
-    protected String configurationFile = "config/scriptengines/docker-compose.properties";
+    private final String configurationFile = "config/scriptengines/docker-compose.properties";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Make this final field static too. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS1170)